qiskit_ibm_runtime/options/options.py

```diff
@@ -148,6 +149,9 @@ def _get_program_inputs(options: dict) -> dict:
         # Add additional unknown keys.
         for key in options.keys():
             if key not in known_keys:
+                warnings.warn(
+                    f"Key {key} not in available options. It will be ignored."
```

Comment:
```suggestion
                    f"Key '{key}' is an unrecognized option. It may be ignored."
```
Since arbitrary keyword arguments can be accepted by design (for testing) I think this should be reworded